project/Mima.scala

```diff
@@ -32,6 +32,19 @@ object Mima {
     ProblemFilters.exclude[A]("scala.meta." + metaType)
 
   val apiCompatibilityExceptions: Seq[ProblemFilter] = Seq(
+    // implicit classes
+    exclude[IncompatibleResultTypeProblem]("package.XtensionDialectTreeSyntax"),
```

Comment:
This looks duplicated, no? What message is being printed here? I am curious why it reports `IncompatibleResultTypeProblem`

Reply:
because of anyval, the ctor returns the type of argument now and not the type of implicit class. the calls do not look duplicated.

Reply:
`exclude[IncompatibleResultTypeProblem]("package.XtensionDialectTreeSyntax")` is twice, no?

Reply:
It's a palindrome from what I see :sweat_smile:

Reply:
i was sleeping when i said i didn't see duplicates 🙂 will remove